localstack/init/ready.d/lambda.sh

```diff
@@ -1,21 +1,18 @@
 #!/bin/bash
 
-FUNCTION_NAME="test-jslib-aws-lambda"
+testdata_folder="/etc/localstack/init/testdata/lambda"
+zip_dir=/tmp/lambda
+mkdir -p "$zip_dir"
 
-# Create a dummy lambda function responding with a static string "Hello World!"
-cat >index.js <<EOF
-exports.handler = async function(event, context) {
-    return "Hello World!";
-}
-EOF
+for file in "$testdata_folder"/*; do
+  function_name=$(basename "$file")
+  function_zip="$zip_dir/$function_name.zip"
+  (cd "$file" || exit; zip "$function_zip" ./*)
```

Comment:
@nickcaballero do we need to have them nested in folders? Could we have direct a list of files?

Reply:
The file structure matches how the zips are created. We can flatten the folders, but it'll make the script a bit more complicated.

Reply:
I would prefer to have a simpler script, rather than a simpler folder structure 👍🏻